README.md

```diff
@@ -2398,8 +2405,139 @@ hazelcastClient.getMap('brothersMap').then(function (mp) {
 });
 ```
 
-## 7.9. Monitoring and Logging
+## 7.8. Performance
+
+### 7.8.2. Near Cache
+
+Map entries in Hazelcast are partitioned across the cluster members. Hazelcast clients do not have local data at all. Suppose you read the key `k` a number of times from a Hazelcast client or `k` is owned by another member in your cluster. Then each `map.get(k)` will be a remote operation, which creates a lot of network trips. If you have a data structure that is mostly read, then you should consider creating a local Near Cache, so that reads are sped up and less network traffic is created.
+
+These benefits do not come for free, please consider the following trade-offs:
+
+- If invalidation is enabled and entries are updated frequently, then invalidations will be costly.
+
+- Near Cache breaks the strong consistency guarantees; you might be reading stale data.
+
+Near Cache is highly recommended for data structures that are mostly read.
+
+#### 7.8.2.1. Configuring Near Cache
+
+The following shows the configuration for the Hazelcast Near Cache.
+
+**Declarative:**
+
+```
+{
+    "nearCaches": [
+        {
+            "name": "myDataStructure",
+            "invalidateOnChange": (false|true),
+            "timeToLiveSeconds": (0..Number.MAX_SAFE_INTEGER),
+            "maxIdleSeconds": (0..Number.MAX_SAFE_INTEGER),
+            "inMemoryFormat": "(object|binary|native)",
```

Comment:
There is no `native` format in Node.js. Do we have this option in configuration? It should be fixed if so.